neural_modelling/src/neuron/models/neuron_model_eprop_adaptive_impl.c

```diff
@@ -94,16 +95,17 @@ state_t neuron_model_state_update(
     REAL psi_temp2 = ((absk(psi_temp1)));
     neuron->psi =  ((1.0k - psi_temp2) > 0.0k)?
     		(1.0k/neuron->b_0) *
-//			0.3k *
+			0.3k * //todo why is this commented?
 			(1.0k - psi_temp2) : 0.0k;
+//	if (neuron->refract_timer){
+//	    neuron->psi = 0.0k;
+//	}
+    neuron->psi *= neuron->A;
 
-    // This parameter is OK to update, as the actual size of the array is set in the
-    // header file, which matches the Python code and aligns memory alocations.
-    // The value here can be reduced to limit the number of synapse state updates
-    // required by the neuron
-    uint32_t total_synapses_per_neuron = 100;
-    uint32_t total_input_synapses_per_neuron = 100; //todo should this be fixed
-    uint32_t total_recurrent_synapses_per_neuron = 100; //todo should this be fixed
+//  This parameter is OK to update, as the actual size of the array is set in the header file, which matches the Python code. 
+//  This should make it possible to do a pause and resume cycle and have reliable unloading of data.
+    uint32_t total_input_synapses_per_neuron = 40; //todo should this be fixed?
```

Comment:
I'm going to merge this into eprop_adaptive now, but a reminder that we discussed making these values into parameters that can be sent in from Python.